lkp-exec/install

```diff
@@ -225,11 +225,15 @@ install_packages() {
 			install_ocfs2_tools "$ocfs2_tools_name"
 		fi
 	fi
+	
+	if [ "$distro" = "exaleap-riscv-linux" ]; then
+	echo "riscv64"
```

Comment:
the indent is not right here, need check

Reply:
> the indent is not right here, need check

Which editor you are using to check indentation?

Reply:
you need add extra tab before echo, so content inside if is indented as others.

Reply:
Updated the file as required.